ws-client/Cargo.toml

```diff
@@ -22,7 +22,7 @@ tokioV02-util = { package="tokio-util", version = "0.3", features = ["compat"],
 
 async-trait = "0.1"
 fnv = "1"
-futures = { version = "0.3.14", default-features = false, features = ["std"] }
+futures = { version = "0.3.14", default-features = false, features = ["std", "async-await"] }
```

Comment:
@maciejhirsz I wonder if this was the reason for not using `select!`, to keep the required features down?

Reply:
Interesting. The `select!` in futures also seems to have some weird requirements. Tokio has a `select!` macro that works identical, but doesn't require any `.fuse()` or `pin_mut!()` calls.